### PR TITLE
First attempt to package kodoja (work in progress)

### PR DESCRIPTION
**Pull request by abaizan**
Using the current latest commit and calling this v0.0.0 as a placeholder.

* [X] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [X] This PR adds a new recipe.
* [X] AFAIK, this recipe **is directly relevant to the biological sciences** (otherwise, please submit to the more general purpose [conda-forge channel](https://conda-forge.org/docs/)).
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

Working on this with @peterjc, not yet ready to be merged.

### CODE REVIEW

**Comment by abaizan** [comment]
Failed as not using the latest kodoja with working hash-bang lines on Python scripts - need to:

1. update the kodoja repository https://github.com/abaizan/kodoja
2. Update the recipe to the new kodoja commit
3. Update SHA256 checksum using ``sha256sum``

**Comment by abaizan** [comment]
The pure python change did not work here,

```
WARNING:conda_build.noarch_python:Don't know how to handle file: database_modules.py.  Omitting it from package.
WARNING:conda_build.noarch_python:Don't know how to handle file: kodoja_search.py.  Omitting it from package.
WARNING:conda_build.noarch_python:Don't know how to handle file: diagnostic_modules.py.  Omitting it from package.
WARNING:conda_build.noarch_python:Don't know how to handle file: kodoja_retrieve.py.  Omitting it from package.
WARNING:conda_build.noarch_python:Don't know how to handle file: kodoja_build.py.  Omitting it from package.
```

**Comment by peterjc** [comment]
CircleCI also failing with ``/bin/sh: kodoja_search.py: not found`` but strangely the macOS tests passed.

The documentation for local testing needs updating now that ``simulate_travis.py`` no longer exists (I was having docker problems using that anyway), see https://github.com/bioconda/bioconda.github.io/issues/11

I probably need to search the recipes for other examples which are "a bunch of Python scripts" with no actual Python package to be installed (nothing which becomes available to import in other Python code)...

File: recipes/kodoja/build.sh [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+mkdir -p "$PREFIX"
```
Comment (mbargull) [inline recipes/kodoja/build.sh]: You'll have to use `$PREFIX/bin` as `$PREFIX` won't be on `PATH` and thus those scripts cannot be found.

Comment (peterjc) [reply recipes/kodoja/build.sh]: Ah! Light bulb moment - thank you.

Does the fact this passed the CircleCI Mac tests indicate a test-framework problem or platform difference?

**Comment by peterjc** [comment]
Green light from CircleCI - is this OK to merge now @mbargull ? Thank you for identifying where this was breaking.

File: recipes/kodoja/build.sh (hunk 1 above)
Comment (mbargull) [reply recipes/kodoja/build.sh]: It also confused me. I think a test framework problem -- I'll investigate, thank for reminding me!

**Comment by mbargull** [comment]
Thank you @abaizan and @peterjc!

File: recipes/kodoja/build.sh (hunk 1 above)
Comment (peterjc) [reply recipes/kodoja/build.sh]: Cross reference https://github.com/bioconda/bioconda-recipes/issues/7312#issuecomment-361541550

**Comment by peterjc** [comment]
Excellent, thank you @mbargull - and congratulations @abaizan - we can now try a kodoja v0.0.2 conda install locally 👍

**Pull request merged**